docs/EN_US/ConfiguringHPCC/ConfiguringHPCC.xml

```diff
@@ -1692,6 +1694,63 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
 
         <?hard-pagebreak ?>
 
+        <para>
+          <emphasis role="bold">To add feature authentication flags to a
```

Comment:
Almost missed it, but these aren't authentication but rather authorization (or access) flags.